src/helpers/trayManager.ts

```diff
@@ -134,29 +134,29 @@ export class TrayManager {
     }
   }
 
-  private stringToUuid (_str:string|undefined):string
+// generate GUID from a string
+// https://stackoverflow.com/questions/105034/how-to-create-a-guid-uuid/66085896#66085896
+  private stringToGUID (str:string):string
```

Comment:
This is much better thanks for cleaning it up. I'll get it tested and merge in a few days.